package-lock.json

```diff
@@ -0,0 +1,1692 @@
+{
```

Comment:
Can you remove this? You should bundle this project with `yarn`.

Reply:
hmm okay i was certain that I ran yarn cypress but I will remove.